### PR TITLE
more on kwoc

### PR DESCRIPTION
**Pull request by proffapt**
**Write path(s) which will be affected by this Pull Request.**
- `/events/kwoc.md`

**Justify your changes or addition.**
Added a few more things we do in KWoC.

**Why do you think it should be documented?**
Some steps were missing on the documentation about how KWoC is organized. Gives us - the organizers - a checklist to check upon along with others getting further details on what all happens in KWoC

### CODE REVIEW

File: events/kwoc.md [hunk 1]
```diff
@@ -11,13 +11,18 @@
 - The following things should be ready (or updated) :-
 
   - Website with registration feature tested internally.
+  - Last year's stats on the website.
```
Comment (harshkhandeparkar) [inline events/kwoc.md:14]: Maybe we could add updating testimonials here?

File: events/kwoc.md [hunk 2]
```diff
@@ -11,13 +11,18 @@
...
   - Student and Mentor Docs of KWoC. Make sure that the content of the doc is updated with the current rules of KWoC(especially rules like Mid Evals and End Evals criteria). If there are any new important changes, do mention them in the docs. 
 
 - A dedicated slack workspace for KWoC. If the old one has reached the maximum free limit, create a new Slack workspace.
-  
+
+- Printing and circulation of posters to every hall's, main building's and nalanda's notice boards.
+
 - Intro Seminar of KWoC should be conducted, a week before the registrations begin.
 
+- If possible approach a few organisations or companies - well before the website goes live - to sponsor KWoC for providing prizes to the top-performers (the number of top-performers might vary based on what we get in the swag). Do NOT term them as `winners` - [KWoC is not a competition](https://github.com/kossiitkgp/mistakes/blob/kwoc-22-mistakes/kwoc20.md#5-reduce-competive-nature-)
```
Comment (harshkhandeparkar) [inline events/kwoc.md:24]:
```suggestion
- If possible approach a few organisations or companies - well before the website goes live - to sponsor KWoC for providing swags/goodies to the top-contributors (the number of top-contributors might vary based on what we get in the swag). Do NOT term them as `winners` - [KWoC is not a competition](https://github.com/kossiitkgp/mistakes/blob/kwoc-22-mistakes/kwoc20.md#5-reduce-competive-nature-)
```

File: events/kwoc.md [hunk 3]
```diff
@@ -44,16 +49,16 @@ It is suggested a group of dedicated people from KOSS handle complaints and quer
 
 - A Github Repo for Technical complaints. For example refer [this](https://github.com/kossiitkgp/kwoc-bugs)
 
-- Twitter and Linkedin of KOSS account
+- Discord, Twitter and Linkedin of KOSS account
```
Comment (harshkhandeparkar) [inline events/kwoc.md:52]:
```suggestion
- Facebook, Instagram, Twitter, and Linkedin of KOSS account
```

Add Discord above along with `- KWoC Slack workspace`.

**Review by harshkhandeparkar — COMMENTED** [review]
A few small suggestions.

File: events/kwoc.md (hunk 1 above)
Comment (proffapt) [reply events/kwoc.md:14]: umm can be a good plan, but for that we need to have testimonials. Do we have previous year's testimonials?

Comment (harshkhandeparkar) [reply events/kwoc.md:14]: "Update" can mean everything from checking for link rot to adding new testimonials if we have any

File: events/kwoc.md (hunk 3 above)
Comment (harshkhandeparkar) [reply events/kwoc.md]: Please add the Discord server point back, but in a separate bullet point above

File: events/kwoc.md (hunk 1 above)
Comment (rajivharlalka) [reply events/kwoc.md:14]: @proffapt We sent a survey form to participants right? Can use that

**Comment by rajivharlalka** [comment]
Looks fine

File: events/kwoc.md (hunk 1 above)
Comment (proffapt) [reply events/kwoc.md:14]: yea, ok makes sense

Comment (proffapt) [reply events/kwoc.md:14]: Done

**Review by harshkhandeparkar — APPROVED** [review]
lgtm

**Pull request merged**